eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -165,7 +165,7 @@ jobs:
           echo "**********Running sample: ${sample}"
           bash -c "$sample"
           status=$?
-          if [[ $status -eq 0 ]]; then
+          if [[ $status -eq 0 || "$sample" == *"client_certificate_credential_sample"* ]]; then
```

Comment:
I'm *really* uncomfortable putting a carveout for a specific sample inside the global live.tests.yml file.

This seems like it should be configurable in the ci.yml file.

Reply:
We can create a work item for the EngSys team to create a mechanism for having that list separated and stored somewhere else, or find some other way for the sample to communicate that it will return 1 and it is ok.
Right now, we need to release Identity, and this blocks the CI.
This functionality is tested. It is not the test, it is sample.
This is the best way to fix this at the moment. I do not want to modify the sample to return 0. I do not want to hardcode path to a certificate.

Reply:
Is there a way to remove the sample from the CI pipeline (remove it from the cmakelists.txt in the overall samples directory)?

Forgive me, but you can't get the sample to work in the CI pipeline, I'd rather see the sample removed from the CI pipeline rather than special case it in the CI pipeline. I believe you can do that by removing the `create_per_service_target_build_for_sample(identity client_certificate_credential_sample)` line from the cmakelists.txt.

And since the sample failure is going to be ignored anyway, executing it in the CI pipeline doesn't really provide any value.

If, in the future you make changes to the sample to get it to run in the CI pipeline, you can add it back.

Reply:
We for sure want to build and fail the CI if build fails. We may run it, but if the sample returns 1, it is ok.

Reply:
I agree that we want to build the sample. 

But we don't want to run it if it is guaranteed to fail to work. And I would MUCH prefer to see us change the AzureBuildTargetsForCI.cmake file (in cmakemodules) to allow a parameter that prevents the file from being added to the ${service}-samples.txt file rather than see the CI pipeline modified.